data/configd-include.json

```diff
@@ -22,6 +22,7 @@
 "interfaces-openvpn.py",
 "interfaces-pppoe.py",
 "interfaces-pseudo-ethernet.py",
+"interfaces-wirelessmodem.py",
```

Comment:
This should not be changed in your commit - looks like something went wrong on your end.

Reply:
Seems to be lost in late conflict resolution, I've manually restored this and kept the mainline in sync